db/migrate/20180307172827_remove_resource_backup.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveResourceBackup < ActiveRecord::Migration
+  def change
+    drop_table :resource_backups
```

Comment:
would be good to left the block definition for the rollback

Reply:
Yeah, the same as we had on UB core a few days ago 😄